### PR TITLE
Changes to support end to end operations of image tag management via CLI

### PR DESCRIPTION
**Pull request by andrebriggs**
- Changes to support tagger name in the CLI. 
- Changes to use real URLs for downloads in CLI
- Changes to change some logging to "debug"
- Changes to use the logging package in the data access layer
- Changes to ensure on-boarding action has correct arguments in CLI
- Changes to requirements.txt 
- Removed some logging

### CODE REVIEW

File: functions/pipeline/shared/db_access/db_access_v2.py [hunk 1]
```diff
@@ -269,4 +270,8 @@ def id_generator(size=6, chars=string.ascii_uppercase + string.digits):
     return ''.join(random.choice(chars) for _ in range(size))
 
 if __name__ == "__main__":
+    #Log to console when run locally
```
Comment (mtarng) [inline functions/pipeline/shared/db_access/db_access_v2.py:273]: Neat!

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 azure-functions==1.0.0a5
 azure-functions-worker==1.0.0a6
 azure-storage-blob==1.4.0
+azure-storage-file
```
Comment (mtarng) [inline requirements.txt]: This doesn't need a version?

File: functions/pipeline/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 azure-functions==1.0.0a5
 azure-functions-worker==1.0.0a6
 azure-storage-blob==1.4.0
+azure-storage-file
```
Comment (mtarng) [inline functions/pipeline/requirements.txt]: this doesn't need a version?

**Review by mtarng — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (andrebriggs) [reply requirements.txt]: Fixed

File: functions/pipeline/requirements.txt (hunk 3 above)
Comment (andrebriggs) [reply functions/pipeline/requirements.txt]: fixed

**Pull request merged**